lib/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       global.$ = $$;
       // 同期がうまくいかないのでコールバックで処理する
       global.jqueryify = require('./response/jqueryify');
-      var app = new App({ wao: that, name: serverConfig.name, port: serverConfig.port });
+      var app = new App({ wao: that, name: serverName || serverConfig.name, port: serverPort || serverConfig.port });
```

Comment:
Line is too long.<br>'serverName' is not defined.<br>'serverPort' is not defined.